data/crac/crac-impl/src/test/java/com/powsybl/openrao/data/cracimpl/InjectionSetpointImplTest.java

```diff
@@ -163,6 +163,17 @@ void applyOnShuntCompensator() {
         assertEquals(2., network.getShuntCompensator("SC1").getSectionCount(), 1e-3);
     }
 
+    @Test
+    void canBeAppliedOnShuntCompensator() {
```

Comment:
good catch. please add a test with setpoint = 2 (canBeApplied = true) and setpoint = 1 (canBeApplied = true)

Reply:
done